### PR TITLE
Add notes regarding migrating from pedantic

### PR DESCRIPTION
**Pull request by srawlins**
Fixes https://github.com/dart-lang/lints/issues/37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,8 +64,20 @@ You can customize the pre-defined lint sets, both to disable one or more of the
 lints included, or to add additional lints. For details see [customizing static
 analysis].
 
+## Migrating from the pedantic package
+
+In migrating from the pre-defined lint sets provided by the [pedantic package],
```
Comment (pq) [inline README.md:69]: Is this delta w/ the *latest* pedantic?  (I can't recall.)  If not, we should probably specify a version?

File: README.md [hunk 2]
```diff
@@ -64,8 +64,20 @@ You can customize the pre-defined lint sets, both to disable one or more of the
...
+you get additional static analysis coverage from 29 additional lints.  There
+are, however, 6 lints which the pedantic package includes which are not
+included in the pre-defined lint sets provided by the lints package:
+`always_declare_return_types`, `prefer_single_quotes`,
+`sort_child_properties_last`, `unawaited_futures`, `unsafe_html`, and
+`use_full_hex_values_for_flutter_colors`. To keep any of these lints enabled,
```
Comment (pq) [inline README.md:75]: Maybe we can point out that `sort_child_properties_last` and `use_full_hex_values_for_flutter_colors` are now in `flutter_lints`?

File: README.md [hunk 3]
```diff
@@ -64,8 +64,20 @@ You can customize the pre-defined lint sets, both to disable one or more of the
 lints included, or to add additional lints. For details see [customizing static
 analysis].
 
+## Migrating from the pedantic package
```
Comment (devoncarew) [inline README.md:67]: `the pedantic package` => `package:pedantic` ?

File: README.md [hunk 4]
```diff
@@ -64,8 +64,20 @@ You can customize the pre-defined lint sets, both to disable one or more of the
...
+
+In migrating from the pre-defined lint sets provided by the [pedantic package],
+you get additional static analysis coverage from 29 additional lints.  There
```
Comment (devoncarew) [inline README.md:70]: nit: I'd just say 'additional' once here

**Review by devoncarew — APPROVED** [review]
👍

File: README.md (hunk 2 above)
Comment (srawlins) [reply README.md]: Done

**Review by parlough — COMMENTED** [review]
This is great to include, thanks! I particularly like to include 
`always_declare_return_types`, `prefer_single_quotes`, and `sort_child_properties_last` and I imagine others who used `pedantic` previously will as well :)

**Pull request merged**